client/components/ReviewsWidget/ReviewsWidget.test.jsx

```diff
@@ -0,0 +1,154 @@
+import ReviewsWidget from './ReviewsWidget';
+import MainHeader from '../MainHeader/MainHeader';
+import ReviewTab from '../ReviewTab/ReviewTab';
+import ReviewPager from '../ReviewPager/ReviewPager';
+import Carousel from '../Carousel/Carousel';
+import { REVIEW_LIST, REVIEW_SUMMARY, REVIEW_PICTURES } from '../../../testUtils/mockData';
+
+const moxios = require('moxios');
+
+const TEST_THEME = {
+  colors: {
+    darkGray: '#222222',
+    middleGray: '#595959',
+    lightGray: '#b1b1b1',
+    paleGray: '#aaaaaa',
+  },
+  fontSize: {
+    extraSmall: '13px',
+    small: '14px',
+    normal: '16px',
+    large: '26px',
+  },
+  fontWeight: {
+    light: '300',
+    regular: '400',
+    boldish: '500',
+    bold: '700',
+  },
+  fontShorthand: {
+    baseMain: '300 16px/150% Roboto',
+    smallMain: '300 13px/150% Roboto',
+    boldMain: '500 13px/150% Roboto',
+    badgeMain: '400 13px/1 Roboto',
+    baseHeading: '300 26px/42px GuardianEgypt, Merriweather, serif',
+  },
+};
+
+const getThemeProviderWrappingComponent = (theme) => ({ children }) => (
+  <ThemeProvider theme={theme}>{children}</ThemeProvider>
+);
+
+export const shallowWithTheme = (component, theme) => shallow(component, {
+  wrappingComponent: getThemeProviderWrappingComponent(theme),
+});
+
+export const mountWithTheme = (component, theme) => {
+  const wrapper = mount(component, {
+    wrappingComponent: getThemeProviderWrappingComponent(theme),
+  });
+
+  return wrapper;
+};
+
+// useful helper: console.log(wrapper.debug());
+describe('ReviewsWidget component rendering tests', () => {
+  it('component renders without crashing', () => {
+    shallow(<ReviewsWidget />);
+  });
+
+  it('should render the inner component: MainHeader', () => {
+    const wrapper = shallowWithTheme(<ReviewsWidget />, TEST_THEME);
+    expect(wrapper.find(MainHeader).length).toEqual(1);
+  });
+
+  it('should render the inner component: ReviewTab', () => {
+    const wrapper = shallowWithTheme(<ReviewsWidget />, TEST_THEME);
+    expect(wrapper.find(ReviewTab).length).toEqual(1);
+  });
+
+  it('should render the inner component: ReviewPager', () => {
+    const wrapper = shallowWithTheme(<ReviewsWidget />, TEST_THEME);
+    expect(wrapper.find(ReviewPager).length).toEqual(1);
+  });
+
+  it('should render the inner component: Carousel', () => {
+    const wrapper = shallowWithTheme(<ReviewsWidget />, TEST_THEME);
+    expect(wrapper.find(Carousel).length).toEqual(1);
+  });
+});
+
+describe('ReviewsWidget initial state tests', () => {
+  it('initial value of rating in state should be 0', () => {
+    const wrapper = shallow(<ReviewsWidget />);
+    expect(wrapper.state('rating')).toEqual(0);
+  });
+
+  it('initial value of activeTab in state should be productReviews', () => {
+    const wrapper = shallow(<ReviewsWidget />);
+    expect(wrapper.state('activeTab')).toEqual('productReviews');
+  });
+
+  it('initial value of isDropdownOpen in state should be true', () => {
```

Comment:
You seem to indicate that you expect true here but are testing for false on line 97.

Reply:
:D fixed, thanks!